cluster/examples/kubernetes/rook-cluster.yaml

```diff
@@ -55,6 +55,18 @@ spec:
 #      podAffinity:
 #      podAntiAffinity:
 #      tolerations:
+  resources:
+# the following requests/limits can be used in the other keys too
```

Comment:
- can you describe what cpu and memory limits are applied in this example? 
- I would move the comment about the other keys below and say something like "see the above example to set limits on the other services, if limits are needed"